src/backend/commands/explain.c

```diff
@@ -3501,6 +3507,31 @@ explain_get_index_name(Oid indexId)
 	return result;
 }
 
+/*
+ * Show prefetch statistics
+ */
+static void
+show_prefetch_info(ExplainState *es, const PrefetchInfo* prefetch_info)
+{
+	if (es->format == EXPLAIN_FORMAT_TEXT)
+	{
+			ExplainIndentText(es);
+			appendStringInfo(es->str, "Prefetch: hits=%lld misses=%lld expired=%lld\n",
+							 (long long) prefetch_info->hits,
+							 (long long) prefetch_info->misses,
+							 (long long) prefetch_info->expired);
+	}
+	else
+	{
+		ExplainPropertyInteger("Prefetch Hits", NULL,
+							   prefetch_info->hits, es);
+		ExplainPropertyInteger("Prefetch Misses", NULL,
+							   prefetch_info->misses, es);
+		ExplainPropertyInteger("Prefetch Expired Requests", NULL,
+							   prefetch_info->expired, es);
```

Comment:
I was not sure that this field is meaningful for users - it is not so easy to explain what it means (unlike other fields.
 But if you think that it should be reported then I will add it.

Reply:
Do I correctly understand that duplicates" means attempts to prefetch page more than once?
It seems to be that now it is more or less expressed algorithmically, i.e. prefetching implementation in heapam/vacuum/pg_preward maintains current prefetch position and should not try to prefetch page more than once?
So presence of duplicates more likely should be treated as bug, rather than something legal which should be reported to the users? Certainly, I understand that there is not any warranty that some query plan will access the same relation ages more than once, for example nested loop join can read inner relation arbitrary number of times... So treating duplicates as errors is definitely not a case. But I still not sure that this information is usable  and can be easily explained in documentation (and all user-visible features has to be documented).

But if we want to provide more information about prefetch, then there is one more prefetch metrics 
which seems to be quite useful for analyze: number of unused prefethc requests.
I.e. if we perform `select * from T where ... limit 100` then query execution can stop before fetching all prefetched
pages.  And this prefetch work done by pagesrver is actually waste of time.

Reply:
> Yes, and that's why I think it'd be mostly useful for our own debugging when working on prefetching. 
And this is actually the reason why I didn't include it in prefetch info for EXPLAIN.
Yes, most likely only Neon developers will look at this info. But in any case it is publicly available for all users. So we will need to document it and somehow explain what are prefetch duplicates and why them arise.

> Isn't that number aggregated in the "Prefetch Expired Requests" metric?

Yes, but how can I get this number for the particular request?

Reply:
In any case - I have added "duplicates" to the information reported by EXPLAIN